Gem/Code/Source/UserSettings/MultiplayerSampleUserSettings.h

```diff
@@ -9,9 +9,32 @@
 #pragma once
 #include <AzCore/EBus/EBus.h>
 #include <AzCore/IO/Path/Path.h>
+#include <Atom/Bootstrap/BootstrapNotificationBus.h>
 
 namespace MultiplayerSample
 {
+    enum VolumeChannel : uint8_t
+    {
+        MasterVolume,
+        MusicVolume,
+        SfxVolume,
+        Max
+    };
+
+    enum SpecularReflections : uint8_t
+    {
+        None,
+        ScreenSpace,
+        ScreenSpaceAndRaytracing
+    };
+
+    enum Msaa : uint8_t
```

Comment:
minor: These enums can be made `enum class`

Reply:
I specifically made VolumeChannel _not_ class so that I could use that `Max` to declare a static array size without having to cast it. I can change the other two though.

Reply:
You could still do enum class with a using for max if you wanted:
`enum class VolumeChannel : uint8_t
{
   Master, 
   Music, 
   Sound
}
using VolumeChannelMax = static_cast<size_t>(VolumeChannel::Sound + 1);`